frostsnapp/README.md

```diff
@@ -0,0 +1,34 @@
+# frostsnapp
+
+This repository serves as a template for Flutter projects calling into native Rust
+libraries via `flutter_rust_bridge`.
+
+## Install
+
+To begin, ensure that you have a working installation of the following items:
+- [Flutter SDK](https://docs.flutter.dev/get-started/install)
+- [Rust language](https://rustup.rs/)
```

Comment:
* [Dart](https://dart.dev/get-dart)
* [just](https://github.com/casey/just) for running commands from justfile
* [ninja](https://ninja-build.org/) for making the flutter app